pkg/client/kubelet.go

```diff
@@ -135,7 +136,43 @@ func (c *HTTPKubeletClient) GetPodStatus(host, podNamespace, podID string) (api.
 }
 
 func (c *HTTPKubeletClient) HealthCheck(host string) (probe.Result, error) {
-	return httprobe.DoHTTPProbe(fmt.Sprintf("%s/healthz", c.url(host)), c.Client)
```

Comment:
The following code duplicates some of what is in DoHTTPProbe but if DOHTTPProbe can be changed to return the body of the response, the duplication can be eliminated